apps/api/src/auth/service/auth.service.ts

```diff
@@ -41,15 +41,39 @@ export class AuthService {
       this.logger.error(`Invalid email address: ${email}`)
       throw new BadRequestException('Please enter a valid email address')
     }
+    try {
```

Comment:
We won't need to use try-catch in here, since the exceptions would be handled properly at HTTP layer.